core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -376,12 +377,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.ALL)
           .build();
+  // Used in alluxio-config.sh and conf/log4j.properties
   public static final PropertyKey LOGS_DIR =
       new Builder(Name.LOGS_DIR)
           .setDefaultValue(String.format("${%s}/logs", Name.WORK_DIR))
           .setDescription("The path under Alluxio home directory to store log files. It has a "
               + "corresponding environment variable $ALLUXIO_LOGS_DIR.")
           .setIgnoredSiteProperty(true)
+          .setIsHidden(true)
```

Comment:
at least the two log properties mention the corresponding env var, but i think it's still better to include the full disclaimer.

Reply:
done